app/controllers/orders_controller.rb

```diff
@@ -6,7 +6,7 @@ def new
   def create
     @order = Order.new(order_params)
     if @order.save
-      redirect_to orders_path, notice: 'Order was created'
+      redirect_to order_done_path(order_id: @order.id), notice: 'Order was created'
```

Comment:
You need to change tests as well, since they explicitly tested redirection to root.